packages/wdio-allure-reporter/tests/suite.test.ts

```diff
@@ -544,14 +594,16 @@ for (const protocol of ['webdriver', 'devtools']) {
             reporter.onSuiteEnd(suiteEnd())
             reporter.onRunnerEnd(runnerEnd())
 
-            const results = getResults(outputDir)
+            const { results } = getResults(outputDir)
+            const screenshotAttachments = results[0].steps[0].attachments.filter(
```

Comment:
please assert length before you get value

Reply:
:+1: